src/internal/components/chart-popover/index.tsx

```diff
@@ -20,7 +20,7 @@ export interface ChartPopoverProps extends PopoverProps {
   title?: React.ReactNode;
 
   /** References the element the container is positioned against. */
-  trackRef: React.RefObject<Element>;
+  trackRef: React.RefObject<HTMLElement | SVGElement>;
```

Comment:
Why are these type changes necessary?

Reply:
In the previous PR we decided to type the element that is passed to `isRtl` to be `HTMLElement | SVGElement`. I applied the same typing to the `getLogicalBoundingClientRect` function which required a type update.

Reply:
I see that this is actually _more_ restrictive now than before, so no problems with this.